src/components/ui/Chart.vue

```diff
@@ -4,8 +4,7 @@
 
 <script>
 import uuid from '../../lib/uuid'
-// import Chart from 'chart.js' // With moment.js
-import Chart from 'chart.js/dist/Chart' // Without moment.js
+import Chart from 'chart.js'
 
```

Comment:
Because the moment is very larger and the all locales will be improted.

I think that wen can import it  when we really need id. :)